src/components/AmountTextInput.tsx

```diff
@@ -0,0 +1,74 @@
+import type {ForwardedRef} from 'react';
+import React from 'react';
+import type {StyleProp, TextInput, TextStyle, ViewStyle} from 'react-native';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+import type TextInputSelection from '@src/types/utils/TextInputSelection';
+import TextInputComponent from './TextInput';
+
+type AmountTextInputProps = {
+    /** Formatted amount in local currency  */
+    formattedAmount: string;
+
+    /** A ref to forward to amount text input */
+    forwardedRef?: ForwardedRef<TextInput>;
```

Comment:
Remove `forwardedRef` and instead add a second argument to AmountTextInput component (check out how it's done in TextInput)